### DISCUSSION TITLE
[Translation] - Monika After Story translation into Russian

### DISCUSSION THREAD

**Discussion by SuperRage1274** [Monika-After-Story/MonikaModDev#7824] [general - translations]
Translation of Monika After Story (MAS) into Russian. Recently there was a conversation with ThePotatoGuy about adding our translation to GitHub. Since we have been translating since the very first versions, and most recently we learned that ThePotatoGuy is looking for translators to translate the mod into Russian, so we have posted our translation here.

Link to the build of the translation: https://drive.google.com/open?id=16C3DyM5dCsKMqcPV9g_xmkQrUOmPc5HQ

**Comment by truesonic53** [comment]
Nice

**Comment by Inui-senpai** [comment]
I'm just sitting here.

**Comment by Akh0r** [comment]
> Translation of Monika After Story (MAS) into Russian. 
Как связаться с вами? Работа над переводом ещё ведётся?

**Comment by ghost** [comment]
@Akh0r 
Есть официальная группа по русскому переводу - https://vk.com/monikaafterstory
Думаю перевод приостановлен.

**Comment by truesonic53** [comment]
@Akh0r Единственная группа, которая до сих пор переводит этот мод - https://vk.com/onthecrack
Остальные группы его бросили.
